### PR TITLE
Add a convenience script for updating submodules

### PR DESCRIPTION
**Pull request by juliawu**
I always need to lookup which commands to run to update the submodules of this repo. This PR adds the commands as a script for convenience.

### CODE REVIEW

File: update_git_submodules.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# limitations under the License.
+
+
+# Convenience script for updating git submodules
+# 
+# Requires: git to be installed
+#
+# Usage: ./update_git_submodules.sh from root directory
+
+git submodule foreach git pull origin master
```
Comment (juliawu) [inline update_git_submodules.sh:23]: Reviewers: To confirm, should we pull from origin master or dc master? (after adding dc as a remote)

**Review by keyurva — APPROVED** [review]
Thanks - this will save time every time!

File: update_git_submodules.sh (hunk 1 above)
Comment (shifucun) [reply update_git_submodules.sh:23]: it should be origin. We don't set up "dc" as remote for the submodules.

**Review by shifucun — APPROVED** [review]
thanks!

**Pull request merged**